src/lib/urls.js

```diff
@@ -30,3 +30,39 @@ export function getCanonicalPath(path) {
   }
   return parts.join('/');
 }
+
+export function copyLinkToClipboard() {
```

Comment:
I think it might be better to give this utility its own file. I'm not sure if it makes sense in this url package.

Reply:
I just thought since it deals with URLs it might be the right place, but happy to move to its own file.